### DISCUSSION TITLE
Countries coverage in European Union region

### DISCUSSION THREAD

**Discussion by Leonaides96** [owid/covid-19-data#2726] [Q&A] [answered]
Hi OWID team,

May I have your confirmation on what are the EU countries cover in the Vaccination dataset and the Vaccination Manufacture dataset?

Wish can have you quick response on this questions 

Thanks and regards, 
Leonaides

**Comment by lucasrodes** [comment, marked as answer]
Hi @Leonaides96,
You can find the list of countries in the following links:

- vaccination dataset: https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/locations.csv
- vaccination manufacturer dataset: https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/locations-manufacturer.csv
